src/firekit.ts

```diff
@@ -1884,14 +1888,74 @@ export class RoarFirekit {
       throw new Error('Grade cannot be empty.');
     }
 
+    // +------------------------------------------------------------------+
+    // | Split out org updates, user record updates, and user doc updates |
+    // +------------------------------------------------------------------+
+    const { addOrgs, removeOrgs, password, ...sendUserData } = userData;
+    const email = _get(sendUserData, 'email', null);
+
+    // If orgs are supplied, update user's admin orgs.
+    if (!_isEmpty(addOrgs)) {
+      const sendAddOrgs = {
+        requesterUid,
```

Comment:
`requesterUid` is retrieved in the cloud function from `request.auth.uid` so it is not necessary to add it to the request data.

```suggestion
```